.github/workflows/pre-merge-checks.yml

```diff
@@ -130,3 +130,34 @@ jobs:
       - run: sam build -t deploy/template.yaml
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  api-tests:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      packages: read
+    defaults:
+      run:
+        working-directory: ./api-tests
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup node and npm
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: ./api-tests/.nvmrc
+          cache: npm
+          cache-dependency-path: api-tests/package-lock.json
+          registry-url: "https://npm.pkg.github.com"
+
+      - name: Install
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.PAT_READ_PACKAGES_SCOPE_CHRIS_WYNNE }}
```

Comment:
This probably shouldn't have your name on it...!

The reads seem to work for the gradle build using `secrets.GITHUB_TOKEN`, but I guess you tried that. If we need configuration at the data-vocab end, then we can reach out to one of the data architects.

In any case, we probably can't run these tests pre-merge yet, we'll need to set up an environment for them to run against (it's no good running against build if they rely on parallel code changes!)

See https://govukverify.atlassian.net/browse/PYIC-5878

Reply:
Yeah, i tried with the GITHUB_TOKEN but no dice, unless I did something else wrong. 
But yeah I guess there's no point running against build - will drop this change.